### DISCUSSION TITLE
デカールが反映しないようにできませんか

### DISCUSSION THREAD

**Discussion by papicochan** [pixiv/vroid-sdk-developers#170] [Q&A] [answered]
Unreal Engineにて、元々のモデルに対してはSet Receive Decalの設定でデカールが反映しないようになっているのですが、VRMモデルをアタッチした状態でデカールの上を歩くと、モデルにデカールが投影されてしまいます。

vroid skeletal meshなどに対してSet Receive Decalのノードをかけてみたのですがうまく設定できませんでした。お知恵を頂きたいです。何卒宜しくお願い致します。
![image](https://github.com/user-attachments/assets/3db6df03-1716-4ebc-b2ae-8dc215f5d3a6)

**Comment by mameoPix** [comment, marked as answer]
ご質問いただき、ありがとうございます。
下図のようにBlueprintを組むことで対応が可能となります。
![image](https://github.com/user-attachments/assets/7923161d-721a-41ef-af50-79bcb264c1db)

`BP_VRoidMannequinRetargetAttachActor`をAttachした際に
内部処理としては、VRM4Uの`BP_VrmPoseCopy`と`MToonAttachActor`をAttachしており
実際のVRMの見た目のSkeletal Mesh Componentは`BP_VrmPoseCopy`が所持しているためです。

**Comment by papicochan** [comment]
ありがとうございました！
まさに必要としていた事ができました。
使用についてもご解説頂き本当にありがとうございます
